typescript/cli/package.json

```diff
@@ -49,9 +50,10 @@
   },
   "scripts": {
     "hyperlane": "node ./dist/cli.js",
-    "build": "yarn version:update && tsc",
+    "build": "yarn version:update && tsc && yarn bundle",
+    "bundle": "rm -rf ./cli-bundle && ncc build ./dist/cli.js -o cli-bundle && node ./scripts/ncc.post-bundle.mjs",
```

Comment:
Why don't we invoke `bundle` from `prepack`? (that would require removing `yarn bundle` from `build` of course)

Reply:
agree that we dont need to bundle in build